localization/twist2accel/launch/twist2accel.launch.xml

```diff
@@ -4,7 +4,7 @@
   <arg name="in_odom" default="in_odom"/>
   <arg name="in_twist" default="in_twist"/>
   <arg name="out_accel" default="out_accel"/>
-  <node pkg="twist2accel" exec="twist2accel" name="twist2accel" output="screen">
+  <node pkg="twist2accel" exec="twist2accel_node" output="screen">
```

Comment:
Similarly, please change this from `screen` to `both`.

Reply:
I changed the log output from "screen" to "both" in https://github.com/autowarefoundation/autoware.universe/pull/7101/commits/c5c4919c2db65dc80d63a1f4e043a11c04c0e54a